uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingrequest/BookingRequestPresenter.kt

```diff
@@ -5,17 +5,14 @@ import androidx.lifecycle.LifecycleObserver
 import androidx.lifecycle.Observer
 import com.karhoo.sdk.api.KarhooError
 import com.karhoo.sdk.api.datastore.user.UserStore
-import com.karhoo.sdk.api.model.FlightDetails
-import com.karhoo.sdk.api.model.LocationInfo
-import com.karhoo.sdk.api.model.Poi
-import com.karhoo.sdk.api.model.Price
-import com.karhoo.sdk.api.model.Quote
-import com.karhoo.sdk.api.model.TripInfo
+import com.karhoo.sdk.api.model.*
```

Comment:
Replace wildcard

Reply:
Android studio doesn't let me - I think because greater than X number of models are used in this class. Each time I try to import models individually it refactors to *

Reply:
There's a setting for AS that you need to update in Preferrences > Editor > Code Style > Kotlin > Imports

Reply:
It is an import issue in the code style. Like what JoJo mentioned above, would be best to go to imports and bump the number of imports it allows before placing a wildcard down up to 99, that way it will never do wildcard imports.